recipe/meta.yaml

```
@@ -24,6 +24,9 @@ source:
     # change tools module location
     - patches/tools.patch
 
+    # https://github.com/pyqtgraph/pyqtgraph/issues/3265
+    - patches/0001-Fix-memory-leak.patch
+
 build:
   number: 0
```

```suggestion
  number: 1
```

you need to bump the build number